README.md

```diff
@@ -145,38 +145,100 @@ $http.get('https://adobeioruntime.net/api/v1/web/helix/helix-services/status@v3/
 );
 ```
 
-## Automated Update of Synthetics Checks
+## Automated Monitoring
 
-`helix-status` provides a small command line tool called `synthetics-check` that enables the automated update or creation of checks in New Relics Synthetics. It is intended to be run as part of your deployment pipeline and allows you to keep the health check format and the extracted custom fields in sync.
+`helix-status` provides the following command line tools intended to be run as part of your deployment pipeline to automate your monitoring:
+
+### Statuspage: Automated Update of Components
+
+`statuspage` allows to automatically create components in Statuspage.
 
 Usage:
 
 ```bash
-$ npx synthetics-check
-synthetics-check <cmd> url
+$ npx statuspage
+statuspage <cmd>
 
 Commands:
-  synthetics-check create url  Create a new check
-  synthetics-check update url  Create or update an existing check
+  statuspage setup  Create or reuse a Statuspage component
 
 Options:
   --version  Show version number                                       [boolean]
   --help     Show help                                                 [boolean]
+  --auth     Statuspage API Key (or env $STATUSPAGE_AUTH)    [string] [required]
+  --page_id  Statuspage Page ID (or env $STATUSPGAGE_PAGE_ID)[string] [required]
+  --name     The name of the component                                  [string]
+  --group    The name of an existing component group                    [string]
+  --silent   Reduce output to automation email only   [boolean] [default: false]
+
+$ npx statuspage setup --group "Delivery"
+Creating component @adobe/helix-status in group Delivery
+Automation email: component+id@notifications.statuspage.io
+done.
+```
+
+You can pre-configure the `name` and `group` arguments in your `package.json`:
+```json
+"statuspage": {
+  "name": "Helix Status",
+  "group": "Delivery"
+}
+```
+By default, the check will use the package `name` from your `package.json`, and leave group empty.
+
+`statuspage` requires a Statuspage [API Key](https://developer.statuspage.io/#section/Authentication) that should be passed using either the `--auth` parameter or the `STATUSPAGE_AUTH` environment variable, as well as a Statuspage [Page ID] that should be passed using either the `--page_id` parameter or the `STATUSPAGE_PAGE_ID` environment variable. 
+
+### New Relic: Automated Update of Synthetics Checks, Alert Policies and Notification Channels
+
+`newrelic` automates the following New Relic features:
+1. creation or update of monitors in New Relics Synthetics
+1. creation of notification channels in New Relic Alerts
+1. creation or update of alert policies in New Relic Alerts
```

Comment:
@rofe you mentioned that New Relic had to enable an API for our account to make this work. It would be worth documenting this here or below.

Reply:
Sure. It's documented here: https://docs.newrelic.com/docs/multi-location-synthetics-alert-conditions

I'll add the link to the readme.